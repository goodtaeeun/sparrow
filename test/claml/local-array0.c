int f(int x) {
  int l[4] = {1, 2, 3, 4};
  return 1;
}
