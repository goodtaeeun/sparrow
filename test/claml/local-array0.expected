/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "local-array0.c"
int f(int x) {
  int l[4];

  {
#line 2
    l[0] = 1;
#line 2
    l[1] = 2;
#line 2
    l[2] = 3;
#line 2
    l[3] = 4;
#line 3
    return (1);
  }
}
